src/components/organisms/tables/LogsSubRowTable/LogsSubRowTable.tsx

```diff
@@ -0,0 +1,109 @@
+import { FC } from 'react'
+import DataTable, {
+  TableSizeTypes,
+} from 'components/organisms/DataTable/DataTable'
+import { createColumnHelper, ColumnDef } from '@tanstack/react-table'
+import classes from './classes.module.scss'
+import { useTranslation } from 'react-i18next'
+import classNames from 'classnames'
+import { includes, join, map, omit, pickBy, toLower } from 'lodash'
+import { EventParameters, ObjectParameters } from 'types/auditLogs'
+
+type AuditLog = {
```

Comment:
**[non-blocker]**

There seem to be at least 2 duplicates for AuditLog type, kind of even 3. Would be good to have just 1, where the actual structure of the AuditLog is defined and in other places you can use Omit or Pick to take only the fields you need from there